virttest/shared/cfg/guest-os/Windows.cfg

```diff
@@ -41,9 +41,11 @@
     # Set verifier flags for windows guests
     Win2016, Win2019:
         windows_verifier_flags = "0x009209bb"
+        windows_verifier_sys = "netkvm ndis"
 
     Win8..1, Win2012..r2:
         windows_verifier_flags = "0x008209bb"
+        windows_verifier_sys = "netkvm ndis"
```

Comment:
We only want to enable netkvm and ndis for netkvm cases, other features' case need to enable their own features' files.
For your changes, all feature will enable netkvm ndis, that's not we expect.

Reply:
Okay, will update the code.